Tasks/PackageDownloader/task.json

```diff
@@ -0,0 +1,110 @@
+{
+  "id": "8d6e8f7e-267d-442d-8c92-1f586864c62f",
+  "name": "PackageDownloader",
+  "friendlyName": "Package Downloader",
+  "description": "Download package from package management.",
+  "helpMarkDown": "",
```

Comment:
can we add help markdown that indicates that the task requires Package Management extension to be installed?